src/Assignment-4/assignment4.css

```diff
@@ -5,3 +5,117 @@
 width: 100%;
 height: 100%;
 }
+body {
+    font-family: "Raleway",sans-serif;
+}
+header{
+    background: #92E0E5;
+    color:white;
+    height:60px;
+    padding: 0px 25px;
+    border-top: 2px solid thistle;
+}
+header img{
+    float: left;
+    
+    height: 50px;
+    width: 50px;
+    margin-top: 8px;
+    object-fit: cover;
+}
+header nav{
+    float: right;
+}
+nav li{
+    display: inline-block;
+    padding: 5px 10px;
+}
+nav li:hover{
+    background: #3F8DA7;
+    cursor: pointer;
+    color:white;
+}
+#hero {
+    background: url("../Assignment-4/images/background\ image.png")no-repeat;
+    background-size: cover;
+    text-align: center;
+    padding:25px 0px;
+    margin-bottom: 10px;
+}
+#hero h2{
+    font-size: 30px;
+    margin-bottom: 15px;
+}
+#hero img{
+    display: block;
+    border-radius: 50%;
+    width: 120px;
+    height: 120px;
+    border: 5px solid #3F8DA7;
+    margin: auto;
+    margin-bottom: 25px;
+}
+#hero a{
+    background: white;
+    border: 5px solid #3F8DA7;
+    text-decoration: none;
+    padding: 5px 10px;
+    color: #3F8DA7;
+}
+#hero a:hover{
+    background: #3F8DA7;
+    color: white;
+}
+.three-col {
```

Comment:
You can try to center the three-col to align with header and footer

Reply:
ok. i will make the above modifications